doc/sphinx-guides/source/admin/dataverses-datasets.rst

```diff
@@ -129,15 +129,21 @@ Lists the link(s) created between a dataset and a Dataverse collection (see the
 
     curl -H "X-Dataverse-key: $API_TOKEN" http://$SERVER/api/datasets/$linked-dataset-id/links
 
-It returns a list in the following format:
+It returns a list in the following format (new format as of v6.4):
```

Comment:
"It was decided today that there is no need for backward compatibility since the api was never documented"

Yes, we rarely document the output JSON, but clients can still rely on it.

Since the change is backward-incompatible, please update the API changelog ( https://guides.dataverse.org/en/latest/api/changelog.html ).

Reply:
added

Reply:
Thanks! Looks great.